scheduler.py

```diff
@@ -12,42 +12,47 @@
 
 raven_client = RavenClient()
 
+logger = logging.getLogger(__name__)
 
 # When testing changes, set the "TEST_SCHEDULE" envvar to run more often
 if os.getenv("TEST_SCHEDULE"):
     schedule_kwargs = {"hour": "*", "minute": "*/10"}
 else:
     schedule_kwargs = {"hour": 4}
 
-logging.basicConfig()
```

Comment:
From https://docs.python.org/3/library/logging.html#logging.basicConfig

> The functions debug(), info(), warning(), error() and critical() will call basicConfig() automatically if no handlers are defined for the root logger.
> This function does nothing if the root logger already has handlers configured for it.

So in the case before we setup slack logging it calls it automatically, and in the case slack logging is set up, it's a no-op.